src/shims/ApiCompatBaseline.netfx.netstandard20.txt

```diff
@@ -0,0 +1,164 @@
+Compat issues with assembly mscorlib:
```

Comment:
The seeds are the RefPath which is correct the issue is that these don't have netfx refs that get bin-placed into the ref\netfx folder. As we work through your support package stuff we can get this baseline down close to zero (there will still be missing members on existing types).